library/webpack-loaders/css-loader.js

```diff
@@ -1,30 +1,34 @@
 const { relative, dirname } = require('path')
+const loaderUtils = require('loader-utils')
 const postcss = require('postcss')
 
 const plugins = [
   // these plugis need to run on each file individual file
   // look at the source of postcss-modules to see that it effectively runs all modules twice
-  ['postcss-plugin-composition', ({ onImport, onExport, resolve }) => [
+  ['postcss-plugin-composition', ({ onImport, onExport, resolve }, options) => [
```

Comment:
I'll go for the latter for now but are curious what the `getClassNamePattern`-implementation would look like?